### PR TITLE
ref: Start using python types from sentry-kafka-schemas

### PR DESCRIPTION
**Pull request by untitaker**
after https://github.com/getsentry/sentry-kafka-schemas/pull/24, we have python types generated from schemas. those schemas are being validated in production at the moment

also requires https://github.com/getsentry/sentry-kafka-schemas/pull/29 to be merged

**NOTE**: Before deploying this, deploy the bump to sentry-kafka-schemas to ensure that the updated event schema passes validation.

### CODE REVIEW

File: snuba/datasets/processors/errors_processor.py [hunk 1]
```diff
@@ -193,6 +218,8 @@ def extract_custom(
         output["type"] = _unicodify(data.get("type", ""))
         output["title"] = _unicodify(data.get("title", ""))
 
+        return contexts
```
Comment (evanh) [inline snuba/datasets/processors/errors_processor.py:221]: Why did you change this return the contexts?

File: snuba/datasets/processors/querylog_processor.py [hunk 2]
```diff
@@ -70,15 +75,16 @@ def __extract_query_list(
...
             consistent.append(int(query["stats"].get("consistent") or 0))
-            profile = query.get("profile") or {
+            fallback_profile: ClickhouseQueryProfile = {
                 "time_range": 0,
                 "all_columns": [],
                 "multi_level_condition": False,
                 "where_profile": {"columns": [], "mapping_cols": []},
                 "groupby_cols": [],
                 "array_join_cols": [],
-                "bytes_scanned": 0,
```
Comment (evanh) [inline snuba/datasets/processors/querylog_processor.py:80]: Why was this removed?

Comment (untitaker) [reply snuba/datasets/processors/querylog_processor.py:80]: because we do not need it later. it also seems that we have not encountered this value in prod when the dict is present

Comment (evanh) [reply snuba/datasets/processors/querylog_processor.py:80]: I believe the plan was to add this value, but the migrations necessary didn't happen. We can always add it back I guess.

**Review by evanh — APPROVED** [review]

File: snuba/datasets/processors/querylog_processor.py (hunk 2 above)
Comment (untitaker) [reply snuba/datasets/processors/querylog_processor.py:80]: bytes scanned is extracted from a different subkey later on, so i have no idea if this was even correct to include in the first place

**Review by lynnagara — APPROVED** [review]
